### ISSUE TITLE
Pylint doesn't recognize methods overriden via assignment

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#97]
Originally reported by: **Grzegorz Niewisiewicz (BitBucket: [grn](http://bitbucket.org/grn))**

---
# Problem

When a method is defined as `method = some_other_method` then Pylint seems to ignore this definition.
# How to reproduce

Save the file below as `test.py`

```
#!python

class BaseClass(object):
    def method(self):
        raise NotImplementedError()

class Subclass(BaseClass):
    def some_method(self):
        print 'Hi!'

    method = some_method
```

and run `pylint --disable=all --enable=abstract-method test.py`. This will produce

```
#!

************* Module test
W:  5, 0: Method 'method' is abstract in class 'BaseClass' but is not overridden (abstract-method)
```
# Expected result

I expect Pylint to treat the code above as if it were defined using `def`. No error should be reported.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/97
